config/config.go

```diff
@@ -857,21 +857,36 @@ func LogPelicanVersion() {
 	log.Infoln("Build Commit:", GetBuiltCommit())
 }
 
-// Print Pelican configuration to stderr
+// Dump Pelican configuration as structured logs and a human-readable format to stderr
 func PrintConfig() error {
 	rawConfig, err := param.UnmarshalConfig(viper.GetViper())
 	if err != nil {
 		return err
 	}
-	bytes, err := json.MarshalIndent(*rawConfig, "", "  ")
+
+	configBytes, err := json.MarshalIndent(rawConfig, "", "  ")
 	if err != nil {
 		return err
 	}
-	fmt.Fprintln(os.Stderr,
-		"================ Pelican Configuration ================\n",
-		string(bytes),
-		"\n",
-		"============= End of Pelican Configuration ============")
+
+	configDump := fmt.Sprintf(
+		"================ Pelican Configuration ================\n%s\n============= End of Pelican Configuration ============",
```

Comment:
To make this a bit easier to read, can you format this with a raw string literal? I'm thinking something like:
```
configDump := fmt.Sprintf(
`====blah====
%s
====end blah====`, string(configBytes))
```

Maybe this is just my preference, if you disagree feel free to disregard.